Description: Skip tests that require writeable /dev/tty0
 * ecore fb
Author: Andreas Metzler <ametzler@debian.org>
Origin: other
Forwarded: no
Last-Update: 2016-01-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/tests/ecore/ecore_test_ecore_fb.c
+++ b/src/tests/ecore/ecore_test_ecore_fb.c
@@ -20,6 +20,8 @@
 // so we can focus on the rest that should work
 /*
    int ret, i, j;
+   fprintf(stderr, "Skipping ecore fb test\n");
+   return(77);
 
    for (i = 1; i <= MAX_ITER; i++)
      {
